src/js/evented.js

```diff
@@ -0,0 +1,71 @@
+export class Evented {
+  constructor(/* options = {}*/) {
```

Comment:
if we're not using constructors or static methods, why use a class? I'd say export a default object.